### PR TITLE
US047 Add tests for viewing Live state of a collection exercise

### PR DESCRIPTION
**Pull request by jcox-dev**
CE's with go-live events in the past in reality should have already been flagged somehow. This botches things somewhat by forcing the scheduler's hand with an alteration (via PUT to /events) to a collection exercise's go-live event.

To test: 
```bash
pipenv install --dev && make start_services system_tests setup && pipenv run behave acceptance_tests/features/view_collection_exercise_live_state.feature
```

### CODE REVIEW

**Comment by jcox-dev** [comment]
[US047: Create and View 'Live' State of a CE [M]](https://trello.com/c/6htalBsL/37-us047-create-and-view-live-state-of-a-ce-m)

File: acceptance_tests/features/view_collection_exercise_live_state.feature [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  Scenario: The 'Live' state is to be displayed when the collection exercise goes live (the 'go live' date hits) and collection instrument is available to respondent
+    Given the user has confirmed that "bricks" "201801" is Ready for Live
+    When "bricks" "201801" go live date hits
+    Then the state of a collection exercise is to be changed to Live
+
+  @us047_s002
+  Scenario: The state can be viewed on the survey details page
+    Given the user is on the Survey Page
+    When they navigate to the "bricks" page
+    Then they are able to see the Status for "201801"
```
Comment (tej99) [inline acceptance_tests/features/view_collection_exercise_live_state.feature]: Maybe, Then they are able to see the **Live** Status for "201801"?

Comment (jcox-dev) [reply acceptance_tests/features/view_collection_exercise_live_state.feature]: Yep, that reads better. These were copied from the acceptance criteria listed in [Confluence](https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/248479826/US047+Create+and+View+Live+State+of+a+CE+M).

File: acceptance_tests/features/steps/view_collection_exercise_live_state.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+@when('"{survey}" "{period}" go live date hits')
+def go_live_date_hits(_, survey, period):
+    # PUT to force the scheduler
+    s_id = {
+        'rsi': '75b19ea0-69a4-4c58-8d7f-4458c8f43f5c',
+        'bricks': 'cb8accda-6118-4d3b-85a3-149e28960c54',
+    }[survey.lower()]
+    datetime = '2018-01-21T00:00:00.000Z'  # some time in the past
+    collection_exercise_controller.update_event_for_collection_exercise(s_id, period, 'go_live', datetime)
+    time.sleep(1)
```
Comment (benjefferies) [inline acceptance_tests/features/steps/view_collection_exercise_live_state.py]: Is it worth putting this in the step which checks for the status and also poll more than once incase it doesn't find it the first time?

Comment (jcox-dev) [reply acceptance_tests/features/steps/view_collection_exercise_live_state.py]: That would be more consistent with the rest of the state-related tests.

**Pull request merged**